runtime-testsuite/test/org/antlr/v4/test/runtime/RuntimeRunner.java

```diff
@@ -26,25 +26,45 @@ public abstract class RuntimeRunner implements AutoCloseable {
 
 	public abstract String getLanguage();
 
-	protected String getExtension() { return getLanguage().toLowerCase(); }
+	protected String getExtension() {
+		return getLanguage().toLowerCase();
+	}
```

Comment:
Please don't change formatting of the existing code without big need. Especially if the request's topic doesn't relate to this change. Here and below.

Reply:
That's just IntelliJ doing it - I am not in to bikeshedding about code formats ;)

Reply:
IntelliJ just did it without me realizing. Formatting is reversed.

Reply:
I suspect it can be disabled in the options.